R/map.R

```diff
@@ -169,13 +169,46 @@ map_ <- function(.type,
   names <- vec_names(.x)
 
   .f <- as_mapper(.f, ...)
-
   i <- 0L
+  print(i)
```

Comment:
For some reason I don't understand in the second run `i` is 5 at this point even though we just assigned 0 to it...